test/e2e/dra/dra.go

```diff
@@ -64,6 +64,10 @@ const (
 	podStartTimeout = 5 * time.Minute
 )
 
+var (
+	extendedResourceName = drautils.ExtendedResourceName
```

Comment:
This variable seems to exist to safe some typing. But I think this indirection just obscures the test code. Please remove the variable and replace with direct usage of `drautils.ExtendedResourceName`.

Reply:
replaced with direct usage of drautils.ExtendedResourceName